packages/components/src/components/tab-header/tab-header.css

```diff
@@ -34,12 +34,12 @@
   box-sizing: border-box;
   align-items: center;
   height: var(--tab-height);
-  padding-right: calc(var(--tab-spacing) / 2);
-  padding-left: calc(var(--tab-spacing) / 2);
-  padding-top: var(--telekom-spacing-unit-x4);
-  padding-bottom: var(--telekom-spacing-unit-x4);
+  padding-right: var(--telekom-spacing-unit-x4);
+  padding-left: var(--telekom-spacing-unit-x4);
+  padding-top: var(--tab-spacing);
+  padding-bottom: var(--tab-spacing);
```

Comment:
I think small should be the default, no? If you think it's OK, let's change it now and we can then roll back if necessary 🙏

Reply:
It was requested to take large as default, since large is the optimized for mobile use case 🙂